lib/utils/qemu.ts

```diff
@@ -37,9 +36,14 @@ export function copyQemu(context: string, arch: string) {
 	const binDir = path.join(context, '.balena');
 	const binPath = path.join(binDir, QEMU_BIN_NAME);
 
-	return Bluebird.resolve(fs.promises.mkdir(binDir))
-		.catch({ code: 'EEXIST' }, function () {
-			// noop
+	return fs.promises
+		.mkdir(binDir)
+		.catch(function (err) {
+			if (err.code === 'EEXIST') {
+				// ignore
+				return;
+			}
+			throw err;
```

Comment:
Personal preference:
```
		.catch((err) => {
			if (err.code !== 'EEXIST') {
				throw err;
			}
		})
```

Reply:
The reason for this style was 1 to be equivalent to the previous style, and 2 because it works better if you have multiple checks, eg
```
if err.code = A then do X
if err.code = B then do Y
else throw err
```
I'm happen to change it if you still prefer it (in the single check case it doesn't matter either way) but that was the reasoning with going for this style